pkg/configs/client/configs.go

```diff
@@ -8,6 +8,7 @@ import (
 	"net/url"
 	"time"
 
+	gklog "github.com/go-kit/kit/log"
```

Comment:
Why use go-kit's logger?

Reply:
Prometheus 2.0 packages use go-kit's logger interfaces everywhere now, so we need to interface with that. See this in the same file below:

```golang
// TODO(prom2): Wrap our logger into a gokit logger.
rule = rules.NewAlertingRule(r.Name, r.Expr, r.Duration, r.Labels, r.Annotations, gklog.NewNopLogger())
```